Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<h3><strong>Challenge</strong> - Let's do it without <code>for</code>  loops - follow the following steps:</h3><ol><li>Calculate a dataframe with the winner (for simplicity, assume the winner is well-defined by the points) of each season. The dataframe should have a Season column and a WinnerTeam column. You can use groupby and <code>apply</code>  - if there is a way to do it without <code>apply</code>  - even better !</li><li>Calculate a dataframe with the goals per season of the home team. It should have the columns: Season, HomeTeam, and TotalHomeTeamGoals</li><li>Repeat step 2 for the Away team</li><li>Merge dataframes from steps 2 and 3 on Season and Team columns, then sum the two goals column. Now we have a dataframe with Season, Team, TotalGoals</li><li>Merge the dataframes from 1 and 4 on season.</li><li>Add a column NextSeason to the dataframe from 5.</li><li>Calculate the final answer !</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>